src/app/controllers/ShipTrackerController.java

```diff
@@ -69,6 +81,46 @@ protected void handleImportShipmentsClick(ActionEvent event) {
         NavigationController.loadPage(NavigationController.IMPORT_SHIPMENTS);
     }
 
+    @FXML 
```

Comment:
ImportShipmentsController.java needs to be created to hold any handler/ActionEvents that aren't the main navigation